src/Tizen.NUI/src/public/BaseComponents/Style/Selector.cs

```diff
@@ -297,6 +301,22 @@ public Selector<T> Clone()
             return cloned;
         }
 
+        /// <summary>
+        /// Clone with type converter.
+        /// </summary>
+        /// <exception cref="ArgumentNullException"> Thrown when converter is null. </exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Selector<TOut> Clone<TOut>(Converter<T, TOut> converter)
+        {
+            if (null == converter) throw new ArgumentNullException(nameof(converter));
+
+            Selector<TOut> result = new Selector<TOut>();
+            result.All = converter(All);
+            result.SelectorItems = SelectorItems.ConvertAll<SelectorItem<TOut>>(m => new SelectorItem<TOut>(m.State, converter(m.Value)));
+
+            return result;
+        }
+
```

Comment:
could you check the build warning below in line 546?
I think null checking for `list` is reasonable.
Or you can add null argument exception.

when you add exception throwing, please add exception comment also.
=> <exception cref=xxx> xxx </exception>
Example: https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69

Reply:
Done!